app.py

```diff
@@ -3,34 +3,48 @@
 from aws_cdk import App, Environment
 
 # Local
-from sds_data_manager.utils.stackbuilder import build_sds
+from sds_data_manager.utils.stackbuilder import build_backup, build_sds
 
 """
 This app is designed to be the dev and production deployment app.
-It defaults to a dev deployment via a default `env` value in cdk.json.
-To deploy to prod, specify `--context env=prod`.
+It defaults to a dev deployment via a default `account_name` value in cdk.json.
+To deploy to prod, specify `--context account_name=prod`.
 To deploy to the backup account (only deploys required backup stacks),
-specify `--context env=backup`.
+specify `--context account_name=backup`.
 
 """
 
 app = App()
 
 # Grab values from context
-sds_region = app.node.try_get_context("sds-region")
-where_to_deploy = app.node.try_get_context("env")
-params = app.node.try_get_context(where_to_deploy)
-
-# Ensure required parameters are present
-if not params or "account" not in params or "sds_id" not in params:
-    raise ValueError("Required context parameters 'account' and 'sds_id' not provided.")
-
-account = params["account"]
-
-env = Environment(account=account, region=sds_region)
-print(f"Deploying to account {account} in region {sds_region}.")
-
-# Deploy SDS resources. This is the default with no CLI context variables set.
-stacks = build_sds(app, env=env, sds_id=params["sds_id"], use_custom_domain=True)
+# account_name is the section we are looking for parameters in
+# within the cdk.json file:
+#    "account_name": {"account": "0123", "region": "us-west-2", ...}
+# This can be overridden via the command line: `--context account_name=prod`
+account_name = app.node.get_context("account_name")
+
+# once we have the account_name, get that section out of cdk.json
+account_config = app.node.get_context(account_name)
+# Add the account_name to the account_config for later lookup
+# and using it within the stacks without needing to pass an extra variable
+account_config["account_name"] = account_name
+account = account_config["account"]
+region = account_config["region"]
+
+env = Environment(account=account, region=region)
+print(f"Using the {account_name} account in the {region} region.")
```

Comment:
I find it helpful to print the account number here too.

Reply:
OK, I thought that would just be more confusing because I have no idea what account numbers belong to what, but I added it back in.